### PR TITLE
Test1(py-taxi-service-startproject)

### PR DESCRIPTION
**Pull request by baranovr**

### CODE REVIEW

**Comment by baranovr** [comment]
Display

File: manage.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```
Comment (Dimosphen1) [inline manage.py:1]: Exclude the DB files from the pull request as mentioned in the checklist.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+venv/
+
+pycahe/
+
+.idea/
+
+*.sqlite3
```
Comment (Dimosphen1) [inline .gitignore:4]: There is no need to add an empty line between each ignore rule in the `.gitignore`, add all rules without empty lines.

File: taxi/admin.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from taxi.models import Manufacturer, Car, Driver
```
Comment (Dimosphen1) [inline taxi/admin.py]: Add a blank line between different groups of imports according to PEP-8. Add this change to all imports that are affected by this issue.

File: taxi/admin.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = ["manufacturer", ]
+    search_fields = ["model", ]
+
+
+admin.site.register(Manufacturer)
```
Comment (Dimosphen1) [inline taxi/admin.py]: Prefer using one approach for registering your models, either using functions or decorators.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,4 @@
+venv/
+pycahe/
+.idea/
+*.sqlite3
```
Comment (viktoria-rybenchuk) [inline .gitignore:4]: add a blank line in the end of the file

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
Remove `db.sqlite3` from your repository

**Pull request closed without merging**